android/src/main/jni/Android.mk

```diff
@@ -5,18 +5,20 @@ include $(LOCAL_PATH)/common.mk
 
 REACT_NATIVE := $(call find-node-module,$(LOCAL_PATH),react-native)
 
-LOCAL_CPPFLAGS := -std=c++17
+LOCAL_CPPFLAGS := -std=c++14
 LOCAL_CPPFLAGS += -fexceptions
 LOCAL_CPPFLAGS += -frtti
 LOCAL_CPPFLAGS += -Wall
 LOCAL_CPPFLAGS += -Wextra
 LOCAL_CPPFLAGS += -Werror
 
+LOCAL_DISABLE_FATAL_LINKER_WARNINGS := true
 LOCAL_LDLIBS := -llog
 
-LOCAL_C_INCLUDES := $(REACT_NATIVE)/ReactCommon/jsi $(LOCAL_PATH)/../cpp
+LOCAL_LDLIBS += -L$(BUILD_DIR)/_libraries/jni/$(APP_ABI)
+
+LOCAL_LDLIBS += -ljscexecutor
```

Comment:
Nice! Suppose this means we're compatible with JSC only right now but that's fine for the time being.

Reply:
yeah that's probably the case.